src/ManagedDripsHub.sol

```diff
@@ -78,6 +78,35 @@ abstract contract ManagedDripsHub is DripsHub, UUPSUpgradeable {
         return super.receiveDrips(user, assetId, maxCycles);
     }
 
+    /// @notice Splits user's received but not split yet funds among receivers.
+    /// @param user The user
+    /// @param assetId The used asset ID
+    /// @param currReceivers The list of the user's current splits receivers.
+    /// @return collectableAmt The amount made collectable for the user
+    /// on top of what was collectable before.
+    /// @return splitAmt The amount split to the user's splits receivers
+    function split(
+        address user,
+        uint256 assetId,
+        SplitsReceiver[] memory currReceivers
+    ) public override whenNotPaused returns (uint128 collectableAmt, uint128 splitAmt) {
+        return super.split(user, assetId, currReceivers);
+    }
+
+    /// @notice Collects user's received already split funds
+    /// and transfers them out of the drips hub contract to that user's wallet.
+    /// @param user The user
+    /// @param assetId The used asset ID
+    /// @return amt The collected amount
+    function collect(address user, uint256 assetId)
+        public
```

Comment:
We will restrict this method with the `msg.sender` check in the upcoming PRs, correct?